bridges/snowbridge/pallets/system/src/lib.rs

```diff
@@ -663,6 +718,31 @@ pub mod pallet {
 			let secondary_exists = Channels::<T>::contains_key(SECONDARY_GOVERNANCE_CHANNEL);
 			primary_exists && secondary_exists
 		}
+
+		pub(crate) fn do_register_token(
+			location: Location,
```

Comment:
To avoid unnecessary cloning.
```suggestion
			location: &Location,
```